elec/src/LMR16030_regulator.ato

```diff
@@ -19,54 +20,130 @@ component LMR16030SDDAR:
     signal GND ~ pin 7
     signal SW ~ pin 8
     signal EP ~ pin 9
+    EP ~ GND  # "Must be connected to GND on PCB"
 
 
 module LMR16030Buck from AdjustableRegulator:
     """
     This follows the configuration here:
     https://www.ti.com/lit/ds/symlink/lmr16030.pdf#page=19
+
+    Usage:
+    - set v_in and v_out
+    - [optinal] set the additional "top-level" params below
+    - replace (->) the inductor and diode with the actual components
+    - set the inductor.value
+    - set the diode.voltage and diode.current
+    - set c_out value
+
     """
+
     # Additional top-level params to configure
-    max_i_current = 3A
+    max_output_current = 3A
     transient_response_300mA_to_3A = 5  # percent
     output_ripple = 50mV
     input_ripple = 400mV
-    switching_frequency = 500kHz
+    switching_frequency = 500kHz ± 10%
+    soft_start_time = 1ms to 5ms
 
     assert v_out < v_in < 60V
-    assert max_i_current < 3A
+    assert max_output_current < 3A
 
     ic = new LMR16030SDDAR
 
+    # Optional: Enable pin
+    # This is normally pulled up, so only connect
+    # to disable by pulling down
+    enable = new Pair
+    enable.io ~ ic.EN
```

Comment:
Pull this up so it's enabled by default?

Reply:
It’s pulled up internally by default, so no need unless we’re trying to modify its behaviour
I should add “internally” to the comment to make that clear